router/CC2652R1/README.md

```diff
@@ -0,0 +1,6 @@
+# LAUNCHXL-CC26X2R1 router firmware
+
+- Router firmware for [LAUNCHXL-CC26X2R1](https://www.ti.com/tool/LAUNCHXL-CC26X2R1) base on **zr_light_CC26X2R1_LAUNCHXL_tirtos_ccs** sample for sdk_4_20_00_35.
+- How to use: It will recognize as **DNCKATSW001** in zigbee2mqtt, work with all available channel, just flash and power the board and allow join. 
+To reset the router, press and hold button **BTN-1** for 10 second and release when led blink.
+- Currently no compilation instructions are available for this firmware.
```

Comment:
Please add these (so also firmwares for CC2652RB and CC1352-P2 can be compiled and maintained)

Reply:
I will add it later. Because three devices need 3 project with additional code changes

Reply:
Hello, are you still planning on sharing instructions on how to compile this FW?

Reply:
@Koenkk I could not share the modification to the firmware but only the hex file. Now i still don't have time for the CC2652RB and CC1352-P2

Reply:
What did you change to make it work on all channels and what to make it also reset with btn1?